lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -332,15 +415,32 @@
         "PVAV with reheat",
         "PVAV with reheat with central air source heat pump",
         "PVAV with PFP boxes",
-        "Residential forced air",
-        "Residential forced air cooling hot water baseboard heat",
-        "Residential forced air with district hot water",
-        "Residential heat pump",
-        "Forced air furnace",
-        "Forced air furnace district chilled water fan coil",
-        "Forced air furnace direct evap cooler",
-        "Residential AC with no heat",
-        "Residential AC with electric baseboard heat"
+        "Residential - no heating or cooling",
+        "Residential - furnace and no cooling",
+        "Residential - furnace and central air conditioner",
+        "Residential - furnace and room air conditioner",
+        "Residential - furnace and evaporative cooler",
+        "Residential - boiler and no cooling",
+        "Residential - boiler and central air conditioner",
+        "Residential - boiler and room air conditioner",
+        "Residential - boiler and evaporative cooler",
+        "Residential - no heating and central air conditioner",
+        "Residential - no heating and room air conditioner",
+        "Residential - no heating and evaporative cooler",
+        "Residential - air-to-air heat pump",
+        "Residential - mini-split heat pump",
+        "Residential - ground-to-air heat pump"
+      ]
+    },
+    "heatingSystemFuelType": {
+      "description": "The fuel type of the heating system.",
```

Comment:
Add a note that this may not apply for certain system types (e.g., heat pumps or electric resistance)?